app/grandchallenge/uploads/widgets.py

```diff
@@ -11,8 +13,12 @@ def __init__(self, *args, allowed_file_types=None, **kwargs):
 
     def get_context(self, *args, **kwargs):
         context = super().get_context(*args, **kwargs)
+        widget_id = context["widget"]["attrs"]["id"]
+        if re.match(r"^\d", widget_id):
+            # id names cannot start with a number in CSS
```

Comment:
Why not prefix everything for consistency? Or just flip it around to `allowedFileTypes{context['widget']['attrs']['id']`?

Reply:
True, prefixing all makes sense.